src/components/program/ProgramEndorsements.jsx

```diff
@@ -0,0 +1,54 @@
+import { useContext } from 'react';
+import { faQuoteLeft } from '@fortawesome/free-solid-svg-icons';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { ProgramContext } from './ProgramContextProvider';
+
+const ProgramEndorsements = () => {
+  const { program: { corporateEndorsements } } = useContext(ProgramContext);
+
+  if (corporateEndorsements && corporateEndorsements.length > 0) {
```

Comment:
Can we change this statement to `corporateEndorsements?.length > 0` and move it inside the return block? So it will look something like:

```javascript
return (
  { corporateEndorsements?.length > 0 && (
    <div className=".."..
    ....
    ....
  )}
);